### ISSUE TITLE
Tax (Vat/Gst) Inclusive vs Exclusive price

### ISSUE THREAD

**Issue by RasheedAZ** [Merrit/unit_bargain_hunter#70]
Hi Merrit. Thank you for making this awesome, useful and beautiful app and releasing it on multiple platform, especially Flathub.

Can you please also add the option to maybe have a checkbox or toggle for Tax inclusive price per item/location?

This is because while most stores might have the tax included depending on your region, other like warehouse type depos or bulk buy stores often have GST excluded which makes them a huge hassle to compare.

Maybe a customisable tax amount can be set in the settings...?

Thanks :)

**Comment by Merrit** [comment]
Thanks for the issue! :)

I'm not sure I get quite what you are going for; could you explain a little more, or maybe sketch out the issue/solution or something similar?

For example, how would this look? Which toggle would there be, where, and what exactly would it change?

**Comment by RasheedAZ** [comment]
![signal-2023-05-19-224745](https://github.com/Merrit/unit_bargain_hunter/assets/16827461/e81bdf11-4039-4bca-a471-d8883ba0a257)
![signal-2023-05-19-224606](https://github.com/Merrit/unit_bargain_hunter/assets/16827461/e47d4654-fee9-4522-8294-6375e495a9b4)

**Comment by Merrit** [comment]
Thanks for the great example! :grinning: 

I will give this some thought, see what I can come up with.

**Comment by Merrit** [comment]
Just an update; I have a couple ideas to implement this, but am a bit busy at the moment. I will take a stab at it when I get some time though. :)

**Comment by Merrit** [comment]
So as I understand it..

If an item does not include taxes in its price, and this toggle is set "Includes tax" -> false.

Is the goal that the app then grabs the tax percentage from the settings and updates the price it shows to include the tax for you?

**Comment by RasheedAZ** [comment]
> 

Yes  I think that how it should work be if the default assumes that the prices entered by the user already have taxes included.

**Comment by Merrit** [comment]
Thanks for clarifying, I will take a look at this shortly. :)

**Comment by Merrit** [comment]
Here's a beta build, please give it a try. :)

[UnitBargainHunter-Linux-Portable.tar.gz](https://github.com/Merrit/unit_bargain_hunter/files/11737749/UnitBargainHunter-Linux-Portable.tar.gz)

**Comment by RasheedAZ** [comment]
Thanks. Didn't realize this was done. Just tried and seems to work great. There's one thing I noticed that can be improved but I'll make a new issue for it.
